midx-write.c

```diff
@@ -1474,8 +1474,7 @@ int midx_repack(struct repository *r, const char *object_dir, size_t batch_size,
 	repo_config_get_bool(r, "repack.usedeltabaseoffset", &delta_base_offset);
 	repo_config_get_bool(r, "repack.usedeltaislands", &use_delta_islands);
 
-	strvec_pushl(&cmd.args, "pack-objects", "--stdin-packs", "--non-empty",
-		     NULL);
+	strvec_push(&cmd.args, "pack-objects");
```

Comment:
How hard do you think would it be to add a regression test to demonstrate the breakage? If it is easy, it might make sense to include it in this PR, if it is not so easy, I am fine with taking this revert as-is.

Reply:
I'm planning to create a regression test when I send this revert upstream.